testplan/report/testing/schemas.py

```diff
@@ -3,18 +3,8 @@
 import functools
 import json
 
-from typing import Mapping
```

Comment:
Why not:
```
try:
    from collections.abc import Mapping, Iterable
except ImportError:
    from collections import Mapping, Iterable
```

Reply:
We know that what we got it is a dict so no need to do match for abc